<!-- -->
<launch>
    <arg name="resolution" default="0.1" />
    <arg name="cell_width" default="500" />
    <arg name="cell_height" default="500" />
    <arg name="points_topic" default="points_lanes" />


	<node pkg="object_map" type="points2costmap" name="points2costmap" output="screen">
        <param name="resolution" value="$(arg resolution)" />
        <param name="cell_width" value="$(arg cell_width)" />
        <param name="cell_height" value="$(arg cell_height)" />
        <param name="points_topic" value="$(arg points_topic)" />
	</node>

</launch>

